SIL.Archiving/Generic/ArchivingFileSystem.cs

```diff
@@ -17,7 +17,7 @@ public static string SilCommonDataFolder
 			{
 				// On Linux we have to use /var/lib (instead of CommonApplicationData which
 				// translates to /usr/share and isn't writable by default)
-				string folder = Platform.IsLinux ? "/var/lib" :
+				string folder = Platform.IsUnix ? "/var/lib" :
```

Comment:
Is this true for Mac?

Reply:
I think so, but maybe @chrisvire can weigh in.

Reply:
On newer mac, `/usr/share` is not even writable with `sudo` because of [SIP](https://support.apple.com/en-us/HT204899).

Reply:
How is SilCommonDataFolder used? If is shared data for all users on the computer?

Application Preferences: ~/Library/Preferences/SIL/<AppName> (this is where App Builders saves preferences)
Shared Data for the current user between applications: ~/Library/Application Support/SIL/<Component>
Shared Data for all users: /Users/Shared/SIL/<Component>

Reply:
To whom shall we address @chrisvire's question?

Reply:
@jasonleenaylor: who can answer?